docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,231 +9,1402 @@
    },
```

Comment:
<p>"In the MvNormal examples we just saw," This is a little confusing because its not the example we just saw, but the one two back</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>It's referring to the last two examples with the MvNormal. </p>

Reply:
Explicitly stating last two examples would be nice then!